Python plotly bar (#4607)
Codecademy/docs: content/plotly/concepts/express/terms/bar/bar.md

> ---
> Title: '.bar()'
> Description: 'Generates a chart representing categorical data with vertical bars.'
> Subjects:
>   - 'Computer Science'
>   - 'Data Science'
>   - 'Data Visualization'
> Tags:
>   - 'Data'
>   - 'Graphs'
>   - 'Libraries'
>   - 'Methods'
> CatalogContent:
>   - 'learn-python-3'
>   - 'paths/computer-science'
> ---

Reviewer: ```suggestion
---
Title: '.bar()'
Description: 'Generates a chart representing categorical data with vertical bars.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Data'
  - 'Graphs'
  - 'Libraries'
  - 'Methods'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

> The **`.bar()`** method in Plotly is part of the Plotly Express high-level interface to Plotly. It generates charts representing categorical data with vertical bars. 

Reviewer: ```suggestion
In Plotly, the **`.bar()`** method is part of the high-level interface named `plotly.express`. It generates charts representing categorical data with vertical bars. 
```

## Syntax

> ```pseudo
> plotly.express.bar(x, y, labels, title, width, height, ...)
> ```
> The parameters are:
> 
> - x: String, integer, series or array-like designating the x-axis data.
> - y: String, integer, series or array-like designating the y-axis data.
> - labels: Dictionary of string keys and string values designating the labels for the axis titles, etc. 
> - title: String designating the chart title.
> - width: Integer designating the chart width in pixels.
> - height: Integer designating the chart height in pixels.

Reviewer: ````suggestion
```pseudo
plotly.express.bar(x, y, labels, title, width, height, ...)
```

- x: A string, integer, series, or array-like object designating the x-axis data.
- y: A string, integer, series, or array-like object designating the y-axis data.
- labels: A dictionary with string keys and string values designating column names and labels for axis titles, hovers, and legend entries.
- title: A string designating the chart title.
- width: An integer designating the chart width in pixels.
- height: An integer designating the chart height in pixels.
````

> **Note:** The ellipsis (...) indicates that there can be additional optional parameters beyond those listed here.

## Example

> Below is an example demonstrating how to use the `.bar()` method to create a simple bar chart visualizing the values of items in categories a, b, c and d:

Reviewer: ```suggestion
Below is an example demonstrating how to use the `.bar()` method to create a simple bar chart visualizing the values of items in categories `a`, `b`, `c`, and `d`:
```

> ```python

Reviewer: ```suggestion
```py
```

import plotly.express as px
fig = px.bar(x=['a', 'b', 'c', 'd'], y=[1, 2, 3, 4], labels={'x':'Category', 'y':'Count'}, title='Simple Bar Chart', height=500, width=250)
fig.show()
```

> Output:

Reviewer: ```suggestion
The above code produces the following output:
```

> ![Output of ploltly.express.bar() method example](https://raw.githubusercontent.com/Codecademy/docs/main/media/plotly-bar-example.png)

Reviewer: ```suggestion
![Output for the above example](https://raw.githubusercontent.com/Codecademy/docs/main/media/plotly-bar-example.png)
```
